WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -542,6 +553,14 @@ - (void)viewDidLayoutSubviews
     [super viewDidLayoutSubviews];
 }
 
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
+    UIViewController *currentViewController = self.selectedViewController;
+    if (currentViewController != nil) {
+        return currentViewController.supportedInterfaceOrientations;
+    }
+    return UIInterfaceOrientationMaskAllButUpsideDown;
```

Comment:
@hassaanelgarem Based on this [Apple docs](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621435-supportedinterfaceorientations):

> The default value is [all](https://developer.apple.com/documentation/uikit/uiinterfaceorientationmask/1623035-all) for the iPad idiom and [allButUpsideDown](https://developer.apple.com/documentation/uikit/uiinterfaceorientationmask/1622957-allbutupsidedown) for the iPhone idiom.

Perhaps calling `super.supportedInterfaceOrientations` would be safer here for forward compatibility?

Reply:
Good point! Fixed in 6973c2b91b4b2e68488b3810f4469b773964945a 👍